pyproject.toml

```diff
@@ -65,7 +65,7 @@ wheel.packages = ["llama_cpp"]
 cmake.verbose = true
 cmake.minimum-version = "3.21"
 minimum-version = "0.5.1"
-sdist.include = [".git", "vendor/llama.cpp/*"]
```

Comment:
I'm not sure why `.git` was included before. I can build the package even when I remote the directory completely before the build.